components/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+import React, { FunctionComponent } from 'react';
+
+interface BannerAsideProps {
+  classname: string;
+  text: string;
+}
+
+const SubmitButton: FunctionComponent<BannerAsideProps> = ({ text, classname }) => {
```

Comment:
Rename to `Button`. That way we can use it for more than just submitting.

Reply:
Updated the button type